src/RewindStreamException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+use Http\Client\Exception;
```

Comment:
The namespace to src/ is `Http\Client\Common\Plugin`

But lets put this exception in `Http\Client\Common\Plugin\Exception`